src/diart/inference.py

```diff
@@ -68,7 +68,11 @@ def __call__(self, pipeline: OnlineSpeakerDiarization, source: src.AudioSource)
         # Stream audio through the pipeline
         source.read()
 
-        return load_rttm(rttm_path)[source.uri]
+        annotations = load_rttm(rttm_path)
+        if source.uri in annotations:
+            return annotations[source.uri]
+        else:
+            return Annotation()
```

Comment:
```suggestion
            return Annotation(uri=source.uri)
```

Keeping the uri for consistency here.